client/src/i18n/fr/enterprise.json

```diff
@@ -15,7 +15,9 @@
       "ENABLE_PREPAYMENTS_LABEL" : "Activer les Cautions",
       "ENABLE_PREPAYMENTS_HELP_TEXT" : "L'activation de cette fonctionnalité permet aux patients de déposer des fonds à la fenêtre de paiement avant d'être facturé. Si le patient a un solde créditeur, ses factures seront automatiquement payées après facturation.",
       "ENABLE_DELETE_RECORDS_LABEL" : "Activer la Suppression d'Enregistrements",
-      "ENABLE_DELETE_RECORDS_HELP_TEXT" : "Cette fonctionnalité permet aux utilisateurs de supprimer des transactions comme une alternative à l'inversion des transactions directement à partir des registres."
+      "ENABLE_DELETE_RECORDS_HELP_TEXT" : "Cette fonctionnalité permet aux utilisateurs de supprimer des transactions comme une alternative à l'inversion des transactions directement à partir des registres.",
+      "ENABLE_PASSWORD_VALIDATION_LABEL" : "",
```

Comment:
these translate keys have no value, do you need them? if not go ahead and remove them.

Reply:
Whoops, you are right.  Guess I forgot to translate these values :(